### DISCUSSION TITLE
Problem about TCN model

### DISCUSSION THREAD

**Discussion by Codelab-X** [LongxingTan/Time-series-prediction#27] [Q&A]
i have tried to apply your models into my data, the RNN results are normal, but the result of TCN seems like a constant in all time.
Do you konw whats the reason? Thanks very much!

**Comment by LongxingTan** [comment]
The only change in your experiments is that you change the model from "RNN" to "TCN"， All others are keep the same?

**Reply by Codelab-X** [reply]
yes, and the parameters are kept as the initial setting

**Reply by Codelab-X** [reply]
<img width="173" alt="image" src="https://user-images.githubusercontent.com/113431539/226843070-faea77d1-5186-4bbc-bc30-04cd2c5925ae.png">
like this

**Reply by LongxingTan** [reply]
Which version do you use, I will check the code if so

**Reply by LongxingTan** [reply]
@chx126 
I have re-run the colab examples, and the result is ok to for both rnn and tcn model.
[https://colab.research.google.com/drive/1LHdbrXmQGBSQuNTsbbM5-lAk5WENWF-Q#scrollTo=_miHOcFzap3D](https://colab.research.google.com/drive/1LHdbrXmQGBSQuNTsbbM5-lAk5WENWF-Q#scrollTo=_miHOcFzap3D)

If you still have this problem, maybe you can send me a toy data to let me figure it out.

**Reply by Codelab-X** [reply]
thks, i will check my code again

**Comment by Huji122** [comment]
can you give you code for learning
